src/components/containers/TransformAccordionDev.js

```diff
@@ -0,0 +1,88 @@
+import PlotlyFold from './PlotlyFold';
+import PlotlyPanel from './PlotlyPanel';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {connectTransformToTrace} from 'lib';
+
+const TransformFold = connectTransformToTrace(PlotlyFold);
+
+class TransformAccordion extends Component {
+  render() {
+    const {
+      fullContainer: {transforms = []},
+      localize: _,
+    } = this.context;
+    const {children} = this.props;
+
+    const transformTypes = [
+      {label: _('Filter'), type: 'filter'},
+      {label: _('Split'), type: 'groupby'},
+      {label: _('Aggregate'), type: 'aggregate'},
+    ];
+
+    const transformBy =
+      this.context.container.transforms &&
+      this.context.container.transforms.map(tr => {
+        let retValue = '';
+        if (tr.groupssrc) {
+          retValue = `: ${tr.groupssrc}`;
+        } else if (tr.targetsrc) {
+          retValue = `: ${tr.targetsrc}`;
+        }
+        return retValue;
+      });
```

Comment:
this will look weird in streambed, because like we've seen, `srcs` look like `~100: 1000`

For every `src` we should at least look in the dataSourceOptions array, and pick out the label associated with the src. In the dev app the label and value happen to be the same, but that won't always be the case:
https://github.com/plotly/react-chart-editor/blob/master/dev/App.js#L34